src/NDS.cpp

```diff
@@ -44,9 +44,17 @@
 #include "DSi_DSP.h"
 #include "ARMJIT.h"
 #include "ARMJIT_Memory.h"
+#include "RetroAchievements/RAClient.h"
+#include <string>
 
 namespace melonDS
 {
+
+    namespace Config {
```

Comment:
Why is a new namespace introduced just for RA configuration? Why are these settings singletons and not per-instance?